app/knowledge_panels.py

```diff
@@ -23,6 +25,11 @@ def hunger_game_kp(
     if value is not None:
         query["value_tag"] = value
         description = f"{value} {hunger_game_filter}"
+    if sec_facet is not None:
+        description += f" {sec_facet}"
+    if sec_value is not None:
+        query[sec_facet] = sec_value
+        description += f" {sec_value}"
```

Comment:
This does not work and won't do anything.
The only case where we could really cross values is the case were one of both facet is country or brand.

So you could handle both those cases (remember: first unify your cases, then process).

And when you have two non "mixable" facets, I would propose to put a link for each facet/value instead.

I would say:
- receive facet name and values as a dict, with None for value if you only have a facet name
- if there is a "country" parameter, remove from dict and use it to filter on country
- if there is a "brand" parameter, remove from dict and use it to filter on brand
- then if you still have two value in dict, generates two panels
- if you have no more value, and got a brand parameter, ask for brand questions
- else generate only one panel

Reply:
@alexgarel I use other approach!